### PR TITLE
Translations of some common disaggregation terms

### PR DESCRIPTION
**Pull request by brockfanning**
Before auditing the accuracy of these translations, some notes about them:
* These are intended to provide only a subset of the possible disaggregation terms that an SDG dataset might use. Obviously we can't account for ALL possible terms, or this file would balloon in size. So the idea here is to translate the most common and general-purpose of the disaggregation terms, relevant to SDGs.
* Any suggestions for additions, removals, and changes are welcome.

Now on to the translation accuracy: These were Google translations. Here are the approvals so far:

- [ ] am
- [x] ar
- [ ] de
- [x] en
- [ ] es
- [ ] fr
- [ ] ru
- [x] zh-Hans

### CODE REVIEW

**Comment by brockfanning** [comment]
@JustusCas @holypeony @SDGArmenia Would love to have your thoughts on this one.

File: translations/ar/data.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+sex: جنس
+female: إناثا
```
Comment (AAmdeen) [inline translations/ar/data.yml]: انثى

File: translations/ar/data.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+male: الذكر
```
Comment (AAmdeen) [inline translations/ar/data.yml]: ذكر

File: translations/ar/data.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+other: آخر
+unknown: غير معروف
+age: الفئة العمرية
```
Comment (AAmdeen) [inline translations/ar/data.yml]: عمر

File: translations/ar/data.yml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+urbanisation: درجة التحضر
+rural: ريفي
+urban: الحضاري
```
Comment (AAmdeen) [inline translations/ar/data.yml]: حضاري

File: translations/ar/data.yml [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+income: الدخل أو الثروة الكمية
```
Comment (AAmdeen) [inline translations/ar/data.yml]: دخل

File: translations/ar/data.yml [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+quintile1: Quintile 1 (أفقر)
```
Comment (AAmdeen) [inline translations/ar/data.yml]: (خماسي 1 (أفقر

File: translations/ar/data.yml [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+quintile2: خماسي 2
+quintile3: خماسي 3
+quintile4: خماسي 4
+quintile5: خمس كوين (أغنى)
```
Comment (AAmdeen) [inline translations/ar/data.yml]: (خماسي 5 (أغنى

File: translations/ar/data.yml [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+disability: حالة العجز
+disabled: الأشخاص ذوو الإعاقة
```
Comment (AAmdeen) [inline translations/ar/data.yml]: أشخاص ذوو إعاقة

File: translations/ar/data.yml [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+nondisabled: أشخاص بدون إعاقة
+education: مستوى التعليم
```
Comment (AAmdeen) [inline translations/ar/data.yml]: تعليم

File: translations/ar/data.yml [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+primary: تعليم ابتدائي
```
Comment (AAmdeen) [inline translations/ar/data.yml]: ابتدائي

File: translations/ar/data.yml [hunk 11]
```diff
@@ -0,0 +1,27 @@
...
+secondary: التعليم الثانوي
```
Comment (AAmdeen) [inline translations/ar/data.yml]: ثانوي

File: translations/ar/data.yml [hunk 12]
```diff
@@ -0,0 +1,27 @@
...
+tertiary: التعليم العالي
```
Comment (AAmdeen) [inline translations/ar/data.yml]: بعد الثانوي

File: translations/zh-Hans/data.yml [hunk 13]
```diff
@@ -0,0 +1,27 @@
+sex: 性别
+female: 女
```
Comment (holypeony) [inline translations/zh-Hans/data.yml]: 女性

File: translations/zh-Hans/data.yml [hunk 14]
```diff
@@ -0,0 +1,27 @@
...
+male: 男
```
Comment (holypeony) [inline translations/zh-Hans/data.yml]: 男性

File: translations/zh-Hans/data.yml [hunk 15]
```diff
@@ -0,0 +1,27 @@
...
+other: 其他
+unknown: 未知
+age: 年龄阶层
```
Comment (holypeony) [inline translations/zh-Hans/data.yml]: 年龄组

File: translations/zh-Hans/data.yml [hunk 16]
```diff
@@ -0,0 +1,27 @@
...
+quintile4: 五分之一
+quintile5: 五分之一（最丰富）
+disability: 残疾状况
+disabled: 残疾人士
+nondisabled: 没有残疾的人
+education: 教育程度
+primary: 小学教育
+secondary: 中学教育
+tertiary: 高等教育
+occupation: 占用
```
Comment (holypeony) [inline translations/zh-Hans/data.yml]: 职业

File: translations/zh-Hans/data.yml [hunk 17]
```diff
@@ -0,0 +1,27 @@
...
+unit: 测量单位
+percent: 百分
+number: 数
+ratio: 比
```
Comment (holypeony) [inline translations/zh-Hans/data.yml]: 比例

File: translations/zh-Hans/data.yml [hunk 18]
```diff
@@ -0,0 +1,27 @@
...
+disability: 残疾状况
+disabled: 残疾人士
+nondisabled: 没有残疾的人
+education: 教育程度
+primary: 小学教育
+secondary: 中学教育
+tertiary: 高等教育
+occupation: 占用
+unit: 测量单位
+percent: 百分
```
Comment (holypeony) [inline translations/zh-Hans/data.yml]: 百分比

File: translations/zh-Hans/data.yml [hunk 19]
```diff
@@ -0,0 +1,27 @@
...
+quintile5: 五分之一（最丰富）
+disability: 残疾状况
+disabled: 残疾人士
+nondisabled: 没有残疾的人
+education: 教育程度
+primary: 小学教育
+secondary: 中学教育
+tertiary: 高等教育
+occupation: 占用
+unit: 测量单位
```
Comment (holypeony) [inline translations/zh-Hans/data.yml]: 计量单位

File: translations/zh-Hans/data.yml [hunk 20]
```diff
@@ -0,0 +1,27 @@
...
+female: 女
+male: 男
+other: 其他
+unknown: 未知
+age: 年龄阶层
+urbanisation: 城市化程度
+rural: 乡村
+urban: 城市的
+income: 收入或财富分位数
+quintile1: 五分之一（最穷）
```
Comment (holypeony) [inline translations/zh-Hans/data.yml]: 第一五分位数（最贫穷）

File: translations/zh-Hans/data.yml [hunk 21]
```diff
@@ -0,0 +1,27 @@
...
+quintile2: 五分之二
```
Comment (holypeony) [inline translations/zh-Hans/data.yml]: 第二五分位数

File: translations/zh-Hans/data.yml [hunk 22]
```diff
@@ -0,0 +1,27 @@
...
+quintile3: 五分之三
```
Comment (holypeony) [inline translations/zh-Hans/data.yml]: 第三五分位数

File: translations/zh-Hans/data.yml [hunk 23]
```diff
@@ -0,0 +1,27 @@
...
+quintile4: 五分之一
```
Comment (holypeony) [inline translations/zh-Hans/data.yml]: 第四五分位数

File: translations/zh-Hans/data.yml [hunk 24]
```diff
@@ -0,0 +1,27 @@
...
+quintile5: 五分之一（最丰富）
```
Comment (holypeony) [inline translations/zh-Hans/data.yml]: 五分位距（最富有）

File: translations/zh-Hans/data.yml [hunk 25]
```diff
@@ -0,0 +1,27 @@
...
+disability: 残疾状况
+disabled: 残疾人士
+nondisabled: 没有残疾的人
+education: 教育程度
+primary: 小学教育
```
Comment (holypeony) [inline translations/zh-Hans/data.yml]: 初等教育

File: translations/zh-Hans/data.yml [hunk 26]
```diff
@@ -0,0 +1,27 @@
...
+secondary: 中学教育
```
Comment (holypeony) [inline translations/zh-Hans/data.yml]: 中等教育

File: translations/zh-Hans/data.yml [hunk 27]
```diff
@@ -0,0 +1,27 @@
...
+urban: 城市的
+income: 收入或财富分位数
+quintile1: 五分之一（最穷）
+quintile2: 五分之二
+quintile3: 五分之三
+quintile4: 五分之一
+quintile5: 五分之一（最丰富）
+disability: 残疾状况
+disabled: 残疾人士
+nondisabled: 没有残疾的人
```
Comment (holypeony) [inline translations/zh-Hans/data.yml]: 非残疾人士

File: translations/zh-Hans/data.yml [hunk 28]
```diff
@@ -0,0 +1,27 @@
...
+education: 教育程度
+primary: 小学教育
+secondary: 中学教育
+tertiary: 高等教育
+occupation: 占用
+unit: 测量单位
+percent: 百分
+number: 数
```
Comment (holypeony) [inline translations/zh-Hans/data.yml]: 数目

**Comment by brockfanning** [comment]
@holypeony @AAmdeen Thank you so much for the help! I finally incorporated it into the pull-request.

**Comment by brockfanning** [comment]
I'm going to close this and suggest a different approach in #70

**Pull request closed without merging**